### PR TITLE
Bug 1152684 - [Rocketbar] Hide ValueSelector on global-search-request

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1152684

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d385abac210c68cce87035209e3d910c2cfb066

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9b03e8f7474dae91685cb429860cc3634df8e531

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=468d341dee57d679b02f9d99b32d96317a4253f5

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=da6c36e917ffe838af818d8bd4495d358a3b4abb

File: apps/system/test/unit/rocketbar_test.js [hunk 1]
```diff
@@ -527,7 +531,9 @@ suite('system/Rocketbar', function() {
         maximize: function() {},
         isMaximized: function() {}
       },
-      isActive: function() { return true; }
+      isActive: function() { return true; },
```
Comment (KevinGrandon) [inline apps/system/test/unit/rocketbar_test.js]: This should probably be switched to use mock_app_window.js in the future.

**Pull request closed without merging**